person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    is_of_age?
+  end
```

Comment:
- [ ] all methods below the `private` keyword will be private, in this case, can_use_services, will also be private. please change this so the method is public instead.
- [ ] it should depend on the parent's permission if the user can use the services if the age is below 18, please implement this as well.